src/twisted/test/test_proto_helpers.py

```diff
@@ -0,0 +1,238 @@
+# Copyright (c) Twisted Matrix Laboratories.
+# See LICENSE for details.
+
+from twisted.trial import unittest
+
+
+
+class DeprecationTests(unittest.TestCase):
+    """
+    Deprecations in L{twisted.test.proto_helpers}.
+    """
+    def test_AccumlatingProtocol(self):
```

Comment:
nitpick: the coding-standard-compliant casing here would be `test_accumlatingProtocol`.

Reply:
Updated, as part of moving these tests to twisted.internet.test.test_testing.